src/service/document-click.js

```diff
@@ -107,11 +107,17 @@ export class ClickHandler {
     this.boundHandle_ = this.handle_.bind(this);
     this.rootNode_.addEventListener('click', this.boundHandle_);
 
+    /** @private {boolean} */
     this.appendExtraParams_ = false;
-
     shouldAppendExtraParams(this.ampdoc).then(res => {
       this.appendExtraParams_ = res;
     });
+
+    /**
+     * Lazy-generated list of A2A-enabled navigation features.
+     * @private {?Array<string>}
+     */
+    this.a2aFeatures_ = null;
```

Comment:
I thought we wanted to avoid calling it a2a?

Reply:
Externally, yes. Internally I think it's fine.